### ISSUE TITLE
usage?

### ISSUE THREAD

**Issue by gliptak** [CodeFX-org/jdeps-wall-of-shame#1]
Could you add a README with pointers usage/results?

Are results intended to be published at http://codefx-org.github.io/jdeps-wall-of-shame/ ?

Thanks

**Comment by nipafx** [comment]
Yes, but the site is too big to be usable and I didn't have the time to investigate another approach.
